inst/extdata/mass-balance-settings.json

```diff
@@ -0,0 +1,100 @@
+{
+    "MassBalancePlots": [
+        {
+            "Name": "Name of mass balance sub-section (optional)",
```

Comment:
Is this name used in the report?

Reply:
If not empty, yes, the name will be displayed in the report as a sub section